pkg/vault/test/foundry/YieldBearingPoolBufferAsVaultPrimitive.t.sol

```diff
@@ -17,189 +16,125 @@ import { IRateProvider } from "@balancer-labs/v3-interfaces/contracts/vault/IRat
 import { ERC4626TestToken } from "@balancer-labs/v3-solidity-utils/contracts/test/ERC4626TestToken.sol";
 import { ArrayHelpers } from "@balancer-labs/v3-solidity-utils/contracts/test/ArrayHelpers.sol";
 
-import { PoolMock } from "../../contracts/test/PoolMock.sol";
-
-import { BaseVaultTest } from "./utils/BaseVaultTest.sol";
+import { BaseERC4626BufferTest } from "./utils/BaseERC4626BufferTest.sol";
 
 // TODO: rename file. Do later so that we can see the diff.
-contract YieldBearingPoolBufferAsVaultPrimitiveTest is BaseVaultTest {
+contract YieldBearingPoolBufferAsVaultPrimitiveTest is BaseERC4626BufferTest {
     using ArrayHelpers for *;
 
-    ERC4626TestToken internal waDAI;
-    ERC4626TestToken internal waUSDC;
-
-    // For two-token pools with waDAI/waUSDC, keep track of sorted token order.
-    uint256 internal waDaiIdx;
-    uint256 internal waUsdcIdx;
-
-    address internal yieldBearingPool;
-
     // The yield-bearing pool will have 100x the liquidity of the buffer.
-    uint256 internal yieldBearingPoolAmount = 10e6 * 1e18;
-    uint256 internal bufferAmount = yieldBearingPoolAmount / 100;
-    uint256 internal tooLargeSwapAmount = yieldBearingPoolAmount / 2;
+    uint256 internal tooLargeSwapAmount = erc4626PoolInitialAmount / 2;
     // We will swap with 10% of the buffer.
-    uint256 internal swapAmount = bufferAmount / 10;
+    uint256 internal swapAmount = bufferInitialAmount / 10;
     // LP can unbalance buffer with this amount.
-    uint256 internal unbalanceDelta = bufferAmount / 2;
+    uint256 internal unbalanceDelta = bufferInitialAmount / 2;
 
     function setUp() public virtual override {
-        BaseVaultTest.setUp();
-
-        waDAI = new ERC4626TestToken(dai, "Wrapped aDAI", "waDAI", 18);
-        vm.label(address(waDAI), "waDAI");
-
-        // "USDC" is deliberately 18 decimals to test one thing at a time.
-        waUSDC = new ERC4626TestToken(usdc, "Wrapped aUSDC", "waUSDC", 18);
-        vm.label(address(waUSDC), "waUSDC");
-
-        (waDaiIdx, waUsdcIdx) = getSortedIndexes(address(waDAI), address(waUSDC));
-
-        initializeBuffers();
-        initializeYieldBearingPool();
+        bufferInitialAmount = erc4626PoolInitialAmount / 100;
```

Comment:
Same; we're already using this to set `swapAmount` and `unbalanceDelta`. Is that correct?